Mathlib/SetTheory/Cardinal/Arithmetic.lean

```diff
@@ -472,6 +476,30 @@ theorem add_nat_le_add_nat_iff {α β : Cardinal} (n : ℕ) : α + n ≤ β + n
 theorem add_one_le_add_one_iff {α β : Cardinal} : α + 1 ≤ β + 1 ↔ α ≤ β :=
   add_le_add_iff_of_lt_aleph0 one_lt_aleph0
 
+lemma add_lt_add_of_lt_of_lt {κ₁ κ₂ μ₁ μ₂ : Cardinal}
+    (hκ : κ₁ < κ₂) (hμ : μ₁ < μ₂) : κ₁ + μ₁ < κ₂ + μ₂ := by
+  rcases le_or_gt ℵ₀ (κ₂ + μ₂) with hinf | hfin
+  · refine add_lt_of_lt hinf ?_ ?_ <;> apply lt_of_lt_of_le <;> solve | assumption | simp
+  · have hfin_ : κ₂ < ℵ₀ ∧ μ₂ < ℵ₀ := add_lt_aleph0_iff.1 hfin
+    apply lt_of_le_of_lt
+    · exact (add_le_add_iff_of_lt_aleph0 (lt_trans hμ hfin_.right)).mpr <| hκ.le
+    · simpa [add_comm] using (add_lt_add_iff_of_lt_aleph0 hfin_.left).mpr hμ
+
+lemma mul_nat_strictMono {n : ℕ} (hneq0 : n ≠ 0) : StrictMono fun κ : Cardinal ↦ n * κ := by
```

Comment:
This should probably be called `nat_mul_strictMono`. Though it'd also be nice if you added `StrictMono fun x ↦ (x · n)` under the name `mul_nat_strictMono`.